src/MusicalScore/Graphical/EngravingRules.ts

```diff
@@ -292,11 +294,13 @@ export class EngravingRules {
         this.wedgeLineWidth = 0.12;
         this.tupletLineWidth = 0.12;
         this.lyricUnderscoreLineWidth = 0.12;
+        this.lyricUnderscoreLineWidthVexflow = 1.3;
         this.systemThinLineWidth = 0.12;
         this.systemBoldLineWidth = EngravingRules.unit / 2.0;
         this.systemRepetitionEndingLineWidth = 0.12;
         this.systemDotWidth = EngravingRules.unit / 5.0;
         this.distanceBetweenVerticalSystemLines = 0.35;
+        this.noteHeadWidth = this.distanceBetweenVerticalSystemLines * 2; // estimated
```

Comment:
This should actually not be needed, as we get the bounding box from Vexflow including the note heads. So you only need to consider the right border of the BB of the end staff entry. When i look at the rendering of the extend, it seems actually slightly too long (could be the 0.7 that you add)
We can still add this amount, but let's use a different/new name for it.

Reply:
the problem is that the lyrics extend positioning is done in MusicSheetCalculator before any VexFlow drawing, so we don't have a bounding box for the notes yet. I checked that with a debugging session as well.
I could decrease the length of the extend. They seemed fine to me, but i'll check.

Reply:
..But how can you position the line and know it's length, when there was no VexFlow layouting?

Reply:
Good question, not sure. Maybe the Vexflow layouting is done by adding notes to voices and formatting them, but the notes aren't drawn yet. In any case, the GraphicalNote doesn't have valid BoundingBox information yet (which i checked by debugging).
But the width of the note head should pretty much always be the same, unless a rest note follows perhaps. The length of the extend also shouldn't be that critical, i think.